DOCUMENTATION.md

```diff
@@ -11,7 +11,7 @@ If you feel something is missing, not clear or could be improved, please don't h
 **Kind**: global namespace  
 
 * [balena](#balena) : <code>object</code>
-    * [.interceptors](#balena.interceptors) : <code>Array.&lt;Interceptor&gt;</code>
+    * [.interceptors](#balena.interceptors) : <code>[ &#x27;Array&#x27; ].&lt;Interceptor&gt;</code>
```

Comment:
I am not sure why this changed, maybe some of the tooling is environment-specific?

Reply:
I think it's due to an updated dependency

Reply:
That's already updated in master, so we can probably drop this after rebasing.